.github/workflows/validate-merge.yml

```diff
@@ -0,0 +1,24 @@
+name: Validate Merge
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  validate-merge:
+    runs-on: ubuntu-18.04
+    
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.14.2
```

Comment:
LGTM in general! I'd just suggest that we run the `validate-merge` action with both the supported versions rather than just one because that's going to be the actual state of the code when the PR gets merged and we wanna make sure that looks good with all the supported versions whereas for the PR branch validation, running with only one of the Go versions should be fine IMO because that is more to verify for the author that their tests pass on their PR as they do locally.